cli/cmd/sync_v1.go

```diff
@@ -42,18 +61,44 @@ func syncConnectionV1(ctx context.Context, cqDir string, sourceSpec specs.Source
 	log.Info().Str("source", sourceSpec.VersionString()).Strs("destinations", destinationStrings).Time("sync_time", syncTime).Msg("Start sync")
 	defer log.Info().Str("source", sourceSpec.VersionString()).Strs("destinations", destinationStrings).Time("sync_time", syncTime).Msg("End sync")
 
-	destClients, err := newDestinationClientsV0(ctx, sourceSpec, destinationsSpecs, cqDir)
+	destCtx, destCancel := context.WithCancel(context.Background())
```

Comment:
why not do a `defer destCancel()` here? Same for `sourceCancel`

Reply:
Yeah good point, I think we can

Reply:
Updated